# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323663575 18000
#      Sun Dec 11 23:19:35 2011 -0500
# Node ID 79c8a524336031a7bb10accc7b8aab2c3c1ce344
# Parent  3781981be535e80d44c85373b8fdaa60ca5cd097
append + to version

* configure.ac (AC_INIT): Version is now 3.5.90+.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -22,17 +22,17 @@ dnl Copyright (C) 1993-2011 John W. Eato
 
 ### Preserve CFLAGS and CXXFLAGS from the environment before doing
 ### anything else because we don't know which macros might call
 ### AC_PROG_CC or AC_PROG_CXX.
 
 EXTERN_CFLAGS="$CFLAGS"
 EXTERN_CXXFLAGS="$CXXFLAGS"
 
-AC_INIT([GNU Octave], [3.5.90], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.5.90+], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="46"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
 OCTAVE_RELEASE_DATE="2011-12-11"
 OCTAVE_COPYRIGHT="Copyright (C) 2011 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
